.circleci/config.yml

```diff
@@ -989,7 +989,14 @@ workflows:
               only:
                 - /^v\d+\.\d+\.\d+(-rc\d+)?$/
       - build-lotus-soup
-      - build-macos
+      - build-macos:
```

Comment:
idk what this is for, is it related to this breaking on your machine?

Reply:
nah. this is what's blocking the binaries to be built for the past few releases and we fixed it and tested with v1.13.0-rc2, so im simply backporting it here so we have binaries ready!